Second training

koharayuki opened the pull request.

koharayuki committed:
Issue4

:calendersを:planに変更

requireメソッドをストロングパラメーターとして使用するため
例)params.require(:モデル名)

koharayuki committed:
Issue5

#15 <%day~%>を<%= day~%>に変更。

<% %>は非表示にしたい処理
例)each文などユーザーに見せる必要がない処理

<%=  ~%>は表示したい処理
例)チャットルームなど、ユーザーに見せたい処理

koharayuki committed:
Issue6

・index.html.erbに”曜日の情報”を出力するための記述を追加

・calendars_controller.rbにビューファイルへ”曜日の情報”を送る記述を追加

koharayuki committed:
Merge branch '発展・Lesson6用' into second_training

koharayuki commented:
動画のURL＜https://gyazo.com/26ab1cd67a8e9339bfac984657947fce＞
画像のURL＜https://gyazo.com/9807defcb2f73411de5638ad59e4ba2b＞

Kentozzz commented on app/controllers/calendars_controller.rb:
```diff
@@ -15,11 +15,11 @@ def create
 
-  def get_week
-    wdays = ['(日)','(月)','(火)','(水)','(木)','(金)','(土)']
+  def getWeek
```
### ◇ スネークケースに修正しましょう
```
def getWeek
```
メソッド名がキャメルケースになっておりますので修正しましょう！

Kentozzz commented on app/controllers/calendars_controller.rb:
```diff
@@ -15,11 +15,11 @@ def create
 
   def plan_params
-    params.require(:calendars).permit(:date, :plan)
+    params.require(:plan).permit(:date, :plan,)
```
### ◇ 余分なカンマの削除
```
params.require(:plan).permit(:date, :plan,)
```
:planの後にある`,`を削除しておきましょう！

Kentozzz commented on app/controllers/calendars_controller.rb:
```diff
@@ -29,12 +29,25 @@ def get_week
+        @day -=7
+      end
+
+      days = { :month => (@todays_date + x).month, :date => (@todays_date+x).day, :plans => today_plans, :wdays => wdays[@day+x]}
```
### ◇ 余分にxを足しているため修正しましょう
```
days = { :month => (@todays_date + x).month, :date => (@todays_date+x).day, :plans => today_plans, :wdays => wdays[@day+x]}
```
46行目で`-7`をしたにも関わらず49行目で再度xを足すと正常に表示できないかと存じます！修正しましょう！

Kentozzz replied:
### ◇ シンボル型に修正しましょう
```
days = { :month => (@todays_date + x).month, :date => (@todays_date+x).day, :plans => today_plans, :wdays => wdays[@day+x]}
```
ハッシュロケットに戻っておりますのでシンボル型に修正しましょう！
※コンフリクト解消時に修正頂く必要がございました。

Kentozzz reviewed (commented):
レビュー依頼ありがとうございます！
コメントしましたので、ご確認をお願いいたします！

修正後はコミット→プッシュして再度レビュー依頼フォームからレビュー依頼をお願いいたします！

koharayuki committed:
コメントの内容の修正

koharayuki committed:
コメント内容の修正

動画のURL＜https://gyazo.com/7b1dd1c08fbb736c5409f5debb774685＞
画像のURL＜https://gyazo.com/e4dce6aee434d1708953efb8fd454dd1＞

TakashiKozuma commented on app/controllers/calendars_controller.rb:
```diff
@@ -29,12 +30,26 @@ def get_week
+      end
+
+
+      days = { :month => (@todays_date + x).month, :date => (@todays_date+x).day, :plans => today_plans, :wdays => wdays[@wdays]}
```
```rb
days = { :month => (@todays_date + x).month, :date => (@todays_date+x).day, :plans => today_plans, :wdays => wdays[@wdays]}
```
こちらの記述ですが、issue1~3で修正していただいたハッシュロケットが元に戻ってしまっているかと存じます。
お手数ですが、このプルリクエストの中で修正をお願致します。

TakashiKozuma reviewed (commented):
レビュー依頼ありがとうございます！
修正点確認致しました。
コメントを追記いたしました。
ご確認お願いいたします。

修正後はコミット・プッシュのし忘れがないかご確認の上、再度フォームからレビュー依頼をお願い致します。

koharayuki committed:
コメント内容の修正②

動画のURL＜https://gyazo.com/ef9b302e70a1becd8fd17c6e0bd0308d＞

画像のURL＜https://gyazo.com/84893ea4501b300762710422c9669a84＞

ushi10 reviewed (commented):
レビュー依頼ありがとうございます。修正点を確認させていただきました。 
こちら、LGTMとさせていただきます。

改めて挙動を確認していただき、問題がない様子でしたらこちらのブランチを忘れずマージしましょう。
マージ後、ローカルのmasterブランチにて忘れずにプルをしましょう。
その後、次のカリキュラムに進みましょう。
